### PR TITLE
US-14 (FIX): Treat QROCM as QROC + introduce placeholders as requested

### PR DESCRIPTION
**Pull request by mackwic**
by default, no placeholders. Use a placeholder only if `${variable#placeholder}` is used

### CODE REVIEW

**Comment by mackwic** [comment]
c'est live sur http://14-creer-une-epreuve-qroc.pix.beta.gouv.fr

File: app/models/challenge/proposals-as-blocks-mixin.js [hunk 1]
```diff
@@ -38,9 +38,21 @@ export default Ember.Mixin.create({
 
     for (let index = 0; index < parts.length; index += 1) {
       let { lastIsOpening, block } = parseInput((lastIsOpening || false), parts[index]);
-      if (block) {
-        result.push(block);
+      if (!block) {
+        continue;
       }
+
+      if (block.input && 1 <= block.input.indexOf('#')) {
```
Comment (jbuget) [inline app/models/challenge/proposals-as-blocks-mixin.js]: pour simplifier & accélérer la compréhension du code, je mettrais `1 <= block.input.indexOf('#')` dans une méthode bien nommée, du style `hasPlaceholder(block.input)`

File: app/models/challenge/proposals-as-blocks-mixin.js [hunk 2]
```diff
@@ -38,9 +38,21 @@ export default Ember.Mixin.create({
...
+
+        const tmp_parts = block.input.split('#');
```
Comment (jbuget) [inline app/models/challenge/proposals-as-blocks-mixin.js]: rename "tmp_parts" into "blockParts" : 
- "tmp" => bad smell
- "xxx_yyy" =/= camelCase

**Comment by twickham** [comment]
I've deployed this PR to http://14-creer-une-epreuve-qroc.pix.beta.gouv.fr. Please check it out

**Pull request merged**